dotnet/src/Extensions/Planning.SequentialPlanner/FunctionViewExtensions.cs

```diff
@@ -16,8 +16,10 @@ internal static string ToManualString(this FunctionView function)
     {
         var inputs = string.Join("\n", function.Parameters.Select(parameter =>
         {
-            var defaultValueString = string.IsNullOrEmpty(parameter.DefaultValue) ? string.Empty : $" (default value: {parameter.DefaultValue})";
-            return $"  - {parameter.Name}: {parameter.Description}{defaultValueString}";
+            var requiredString = parameter.IsRequired ? "required" : string.Empty;
+            var defaultValueString = string.IsNullOrEmpty(parameter.DefaultValue) ? string.Empty : $"default value: {parameter.DefaultValue}";
+            var detailsString = $" ({AddComma(requiredString, defaultValueString)})";
```

Comment:
Could `string.Join` accomplish the same thing?

Reply:
Nah string.Join doesn't consider if an element is non-empty without an explicit check

![image](https://github.com/microsoft/semantic-kernel/assets/125500434/bb8652e1-319d-40d1-8693-90818f19762b)

fyi @adrianwyatt